core/sphinx/internal/crypto/crypto.go

```diff
@@ -166,8 +169,8 @@ func (k *PacketKeys) Reset() {
 // HOWEVER if privateKeySize is set to zero (0) then the BlindingFactor
 // will not be generated by the KDF. The reason we do this is because
 // KEMSphinx doesn't need the BlindingFactor.
```

Comment:
this comment isn't accurate anymore, it is now based on if `scheme != nil`

Reply:
resolved